adminer/create.inc.php

```diff
@@ -44,13 +44,21 @@
 			if ($field["field"] != "") {
 				if (!$field["has_default"]) {
 					$field["default"] = null;
+				} elseif (($field["has_default"] == 'STORED') || ($field["has_default"] == 'VIRTUAL')) {
+					$field["virtual"] = "GENERATED ALWAYS AS ({$field['default']}) {$field['has_default']}";
```

Comment:
Php manual: `echo "This is wrong: $arr[foo]";`
fix in new commits.

Reply:
I co-created PHP manual and it's not true.

Reply:
http://php.net/manual/en/language.types.string.php
```
// This is wrong for the same reason as $foo[bar] is wrong  outside a string.
// In other words, it will still work, but only because PHP first looks for a
// constant named foo; an error of level E_NOTICE (undefined constant) will be
// thrown.
echo "This is wrong: {$arr[foo][3]}"; 
```

Reply:
You need to read more carefully. `$foo[bar]` is wrong outside a string, not inside a string. `"{$foo[bar]}"` is also wrong but there's nothing wrong on `"$foo[bar]"` because `bar` is treated as a string, not as a constant.